src/dos/program_mousectl.h

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
This one might have slipped the clang reformatting pass (using a lot of space-indenting) Here's what I get when reformat:

![2022-10-21_14-15](https://user-images.githubusercontent.com/1557255/197292981-6819c601-eb11-4eef-9e3c-64517dc42245.png)

Reply:
OK, reformatted.